### ISSUE TITLE
Investigate print css to get graph centered in what is printed

### ISSUE THREAD

**Issue by kdahlquist** [dondi/GRNsight#629]
I just closed #365 because that was fixed, but as @NAnguiano noted, there is actually a problem with what is printed; the graph is cut off at the side.

**Comment by johnllopez616** [comment]
![image](https://user-images.githubusercontent.com/21348600/40568868-3742a1d6-6032-11e8-8432-519bf2961feb.png)

It fits the page now, but it makes the nodes and the menu significantly smaller. Would you like those features changed as well?

Additionally, I don't even really see the functional need for the menu there...that can be removed from the print menu through CSS, however. What do you think?

**Comment by kdahlquist** [comment]
@johnllopez616, agreed, we don't want the sidebar menu included in the print.  We also don't want the D-pad/zoom either if we can get rid of it.

I just tried printing off of both Firefox and Chrome.  In Chrome, the graph was completely on the printed page, but it wasn't for Firefox.

Let's see if getting rid of the sidebar and D-pad improves things.

**Comment by johnllopez616** [comment]
![image](https://user-images.githubusercontent.com/21348600/40737103-9f27cdba-63f4-11e8-8392-88ece1668515.png)

Here's what I got from my Firefox. We will compare it to your computer's Firefox in the next PR.

**Comment by kdahlquist** [comment]
I'm running Firefox v60.0.1 64-bit.  If that's what you have, hopefully it will look the same.

**Comment by kdahlquist** [comment]
I'm going to mark this one good enough for now.  The D-pad and sidebar menu are gone.  I still see a phenomenon where the graph is cut off on the right.  I discovered that I can go into the Firefox page setup settings and select "shrink to fit page width" and then the graph will print smaller and nothing will be cut off.  However there is a blank second page when I do this.  Also weirdly, after printing the same graph 7 or 8 times at different zoom scale, the page header, sidebar, etc. came back.  I reloaded the page in the browser and that fixed that.

Anyway, I declare this good enough for now.  I'm going to stick a backlog label on this in case we want to revisit it in the future.  But we don't have time to continue messing with this now.
